metis-pattern-analysis/src/test/resources/europeana_record_with_P6.xml

```diff
@@ -62,10 +62,7 @@
   </ore:Proxy>
```

Comment:
:heavy_check_mark: non meaningful title, actually is missing, should add a 2 character title just as the specification?

Reply:
It's not missing, otherwise the tests would fail..
Check again L:72

Reply:
thanks didn't saw it :sweat_smile: